src/components/Calculator.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/prefer-stateless-function */
+
+import React from 'react';
+
+class Calculator extends React.Component {
+  render() {
+    return (
+      <>
+        <div className="container">
+          <form>
+            <input type="text" />
+          </form>
+          <div className="keypad">
+            <div className="clear num">AC</div>
+            <div className="ad num">+/-</div>
+            <div className="modulo num">%</div>
+            <div className="divide operator">&divide;</div>
+            <div className="num">7</div>
+            <div className="num">8</div>
+            <div className="num">9</div>
+            <div className="times operator">&times;</div>
+            <div className="num">4</div>
+            <div className="num">5</div>
+            <div className="num">6</div>
+            <div className="clear operator">&ndash;</div>
+            <div className="num">1</div>
+            <div className="num">2</div>
+            <div className="num">3</div>
+            <div className="add operator">+</div>
+            <div className="zero">0</div>
+            <div className="num">.</div>
+            <div className="equals operator">=</div>
+          </div>
```

Comment:
- [ ] Instead of displaying the numbers as the text, I would recommend making them as the buttons. For example:

`<button type="button" className="calc-btn">{text}</button?` 

You can implement this in another file and import it to the component/calculator.

![image](https://user-images.githubusercontent.com/55635977/158366282-728369ce-77ab-4e00-aa4d-5af8badf0294.png)

Reply:
This does look better. thank you @Emmyn5600 I will change it to this